From linux-usb-devel-bounces@lists.sourceforge.net Mon Oct 29 13:41:07 2007
From: Phil Endecott <spam_from_usb_devel@chezphil.org>
Date: Mon, 29 Oct 2007 20:40:30 +0000
Subject: USB: Fix signr comment in usbdevice_fs.h
To: "Alan Stern" <stern@rowland.harvard.edu>
Cc: linux-usb-devel@lists.sourceforge.net
Message-ID: <1193690430304@dmwebmail.japan.chezphil.org>


This trivial documentation patch corrects a comment in usbdevice_fs.h; it
previously suggested that the signal would only be sent on error, but I am
told that it is sent on both successful and unsuccessful completion, and
that zero indicates that no signal should be sent.

Signed-off-by: Phil Endecott <spam_from_usb_devel@chezphil.org>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>

---
 include/linux/usbdevice_fs.h |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

--- a/include/linux/usbdevice_fs.h
+++ b/include/linux/usbdevice_fs.h
@@ -102,7 +102,8 @@ struct usbdevfs_urb {
 	int start_frame;
 	int number_of_packets;
 	int error_count;
-	unsigned int signr;  /* signal to be sent on error, -1 if none should be sent */
+	unsigned int signr;	/* signal to be sent on completion,
+				  or 0 if none should be sent. */
 	void *usercontext;
 	struct usbdevfs_iso_packet_desc iso_frame_desc[0];
 };
